doc/readme/ABOUT.md

```diff
@@ -4,14 +4,14 @@ Shogun is an open-source machine learning library that offers a wide range of ef
 
 * Supports many languages (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and platforms (Linux/Unix, MacOS and Windows) and integrates with their scientific computing environments.
 
-#### Shogun is state-of-the-art
+#### Shogun is a state-of-art
 
-* Efficient implementation (from standard to cutting edge algorithms), modern software architecture in C++.
+* Efficient implementation (from standard to cutting edge algorithms) and modern software architecture in C++.
 * Easy combination of multiple data representations, algorithm classes and general purpose tools for rapid prototyping of data pipelines.
 
 #### Shogun is open source
 
 * Free software, community-based development and machine learning education.
-* Commercially friendly BSD 3-clause license
+* Commercial friendly BSD 3-clause license
```

Comment:
also this was correct imo

Reply:
I think not, can you explain why do you think it's correct?

Reply:
Well, in this case you should use the adverb because you are qualifying an adjective (friendly)